### ISSUE TITLE
Compare issue 'Behavior'

### ISSUE THREAD

**Issue by azza39925** [cuckoosandbox/cuckoo#1494]
So when trying to compare results in the web interface it will come up with error 'behavior' and no other options. I am trying to compare two files that are the same.

**Comment by jbremer** [comment]
Yeah, that's currently not supported very well. We intend to address it properly in a few months or so ;-)
